### PR TITLE
Corey Zaher's Code Review

### PR DESCRIPTION
**Pull request by czaher**
PuzzlePic Code Review 1

- PuzzlePic is a mobile application that transforms a picture, either provided by the user or from an online database, into a sliding puzzle game. 
- PuzzlePic was available and accessible to me on GitHub on time.
- PuzzlePic is understandable and well-documented.
- The Program does compile and displays their main activity.

My Own Commits

- [cc46c5f](https://github.com/zahercj/PictureThis/commit/b3d72930fc9f864b5e7d6a04de0b1683f77c0b35)
- [dbdbc86](https://github.com/zahercj/PictureThis/commit/cc46c5fb37fa5cfbe511359367be0b7b7e7d1260)
- [b136879](https://github.com/zahercj/PictureThis/commit/27eeda301b26782b0f8c064f2de4d98b186a6b5f)

Technical Suggestions

1. I suggest that the group implements regular code reformatting based on a supported Kotlin style guide. This will help make code more consistent and readable throughout your repository. 
[Ref. Kotlin Coding Conventions - Style Guide](https://kotlinlang.org/docs/coding-conventions.html#configure-style-in-ide)
2. I suggested renaming `val rnds` to something more descriptive of it's usage. I refactored all usages of this variable name to randomValue
[Ref. Kotlin Coding Conventions - Choose Good Names](https://kotlinlang.org/docs/coding-conventions.html#choose-good-names)
3. I suggested removing a redundant/second declaration of the `randomURL` variable and added documentation clarifying this change. This is a redundant syntactic element
[Ref. Kotlin Coding Conventions - Avoid Redundant Constructs](https://kotlinlang.org/docs/coding-conventions.html#avoid-redundant-constructs)

What I Learned

> Through reviewing the PuzzlePic code and referencing the Kotlin Coding Conventions documentation, I learned several key concepts. Firstly, I learned how to implement code styling guides through Android Studio. Previously, I had only used node packages like Prettier and ESLint to lint my code. The selection of built-in styling guides to IntelliJ is accessible and convenient. Additionally, I was refreshed on the most proper naming conventions for variables and classes, and reminded of the need to reduce redundancy and syntactical clutter within code.

### CODE REVIEW

**Comment by czaher** [comment]
I added the Android Studio Kotlin style guide to the editor and applied the reformat to all project files. This is suggested by the Kotlin org and will help the team keep consistent coding conventions.

[Ref. Kotlin Coding Conventions - Style Guide](https://kotlinlang.org/docs/coding-conventions.html#configure-style-in-ide)

**Comment by czaher** [comment]
Gave the `var rnds` variable to a more accurate name according to the Kotlin Coding Conventions guidelines.
[Ref. Kotlin Coding Conventions - Choose Good Names](https://kotlinlang.org/docs/coding-conventions.html#choose-good-names)

**Comment by czaher** [comment]
Removed a redundant/second declaration of the `randomURL` variable and added documentation clarifying this change. This is a redundant syntactic element
[Ref. Kotlin Coding Conventions - Avoid Redundant Constructs](https://kotlinlang.org/docs/coding-conventions.html#avoid-redundant-constructs)

File: app/src/main/java/com/example/puzzlepic/MainActivity.kt [hunk 1]
```diff
@@ -11,8 +11,8 @@ class MainActivity : AppCompatActivity() {
         setContentView(R.layout.main_activity)
         if (savedInstanceState == null) {
             supportFragmentManager.beginTransaction()
-                    .replace(R.id.container, MainFragment.newInstance())
-                    .commitNow()
+                .replace(R.id.container, MainFragment.newInstance())
+                .commitNow()
         }
```
Comment (discospiff) [inline app/src/main/java/com/example/puzzlepic/MainActivity.kt:16]: For reformatting... 
1) A caution.  Significant reformats can make future commit diffs difficult, as git has to deal with significant reordering of the logic.
2) The IDE can do this automatically, and you can provide a template for the IDE to use.  Ensure all developers are using this same template, or the one that comes by default with the IDE, and have them reformat before committing.  This ameliorates issue 1, above.

**Comment by discospiff** [comment]
This review is mostly centered around reformatting.  The IDE can format for you.  In lieu of merging this branch, I recommend that the group agree to a common format, and have the IDE implement that format.

**Comment by erswagner7** [comment]
Decided not to use code review suggestions

**Pull request closed without merging**